src/main/java/de/mediathekview/mserver/crawler/CrawlerManager.java

```diff
@@ -485,6 +486,8 @@ private void initializeCrawler(final MServerConfigManager rootConfig) {
         Sender.SR, new SrCrawler(forkJoinPool, messageListeners, progressListeners, rootConfig));
     crawlerMap.put(
         Sender.ZDF, new ZdfCrawler(forkJoinPool, messageListeners, progressListeners, rootConfig));
+    crawlerMap.put(
```

Comment:
Livestream als eigenen Crawler? wäre es nicht sinnvoller, pro Sender die Logik für Livestreams anzusiedeln?

Reply:
Zu den Punkten,
*handleHttpError 
Mein lieblingsatz: "Bei mir kompelierts" 
Im ernst...schaue ich mir an wenn die anderen Kompilefehler gefixt sind (siehe oben)
* SWR 
Da war ein typo. Ist jetzt korrigiert. Ggf muss ich mal eine Testfall drauf setzen.
* Eigener Crawler
Ich denke so hat man weniger Request, da fast alle in einem kommen
und hat somit dann auch weniger Wartung
und bleibt in der alten Strukur mit dem "eigenem" File